app/client/components/GridView.js

```diff
@@ -70,7 +70,8 @@ const SHORT_CLICK_IN_MS = 500;
 // size of the plus width ()
 const PLUS_WIDTH = 40;
 // size of the row number field (we assume 4rem)
-const ROW_NUMBER_WIDTH = 52;
+const ROW_NUMBER_WIDTH_REM = 4;
+const ROW_NUMBER_WIDTH_PX = 3.25 * ROW_NUMBER_WIDTH_REM; // 52px when no text zoom is applied
```

Comment:
Hi @hexaltation.

Can extend the comment here with the math behind it? How 4 * 3.25 * root font size equals 52px ?

Reply:
Hello @berhalak

I pushed a more (I hope) explicit comment.

The problem is I don't know why a factor of 3.25 have to be applied, except that 52px is the working value in `InitialOffset` and `MiddleOffset`.

May be it's better to get rid of `ROW_NUMBER_WIDTH_PX` and make the 3.25 factor appear by magic in Offsets formulas.

```suggestion
const initialOffset = -this.frozenWidth() - 3.25 * ROW_NUMBER_WIDTH + this.width() - revealWidth - PLUS_WIDTH;
```

```suggestion
const middleOffset = -this.frozenWidth() - 3.25 * ROW_NUMBER_WIDTH_PX + this.width() / 2;
```

Let me know